.github/actions/poetry/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Poetry'
+description: 'Install python poetry'
+
+outputs:
+  version:
+    description: "Installed version"
+    value: ${{ steps.poetry-version.outputs.version }}
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        curl -sSL https://install.python-poetry.org | python
```

Comment:
I would suggest to lock the `poetry` version:
```
curl -sSL https://install.python-poetry.org | python3 - --version 1.2.0
curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.2.0 python3 -
```
In the past we had many problems with `poetry` breaking after an update.